package_io/input_parser.star

```diff
@@ -24,6 +24,54 @@ HIGH_DENEB_VALUE_FORK_VERKLE = 20000
 
 ATTR_TO_BE_SKIPPED_AT_ROOT = ("network_params", "participants")
 
+def get_args_from_parsed_input_results(result):
```

Comment:
can this just call `parse_input` inernally?

Reply:
yes it can, I'll do it